tests/run_test.sh

```diff
@@ -80,9 +80,19 @@ cp ${PATHRT}/modules.fv3_${COMPILE_NR}             modules.fv3
 # Get the shell file that loads the "module" command and purges modules:
 cp ${PATHRT}/../NEMS/src/conf/module-setup.sh.inc  module-setup.sh
 if [[ $FV3 = 'true' ]]; then
```

Comment:
Shouldn't this if test be:
`if [[ $WRITE_DOPOST = 'true. ]]; then`

Reply:
Well, I guess the original design might be that test with quilting on support grids can run inline post with those files presented. We can have a separate fix for the rt script issue to remove these files from the non-post tests in current RT.

Reply:
If `WRITE_DOPOST` is not true none of these files should be copied to the run directory, regardless on whether QUILTING is true or false, or whether $FV3 is true.